maven-plugin/src/main/java/org/openmrs/maven/plugins/Build.java

```diff
@@ -174,6 +174,11 @@ private File createTempReactorProject(Server server) throws MojoFailureException
         Model tempModel = createModel();
         Set<Project> watchedModules = server.getWatchedProjects();
         for(Project module: watchedModules){
+            if (!module.getModel().getPomFile().exists()) {
+                throw new IllegalStateException("Source code of module " + module.getArtifactId() + " doesn't exist. " +
```

Comment:
Please mention path e.g. "Module " + module.getArtifactId() + " cannot be found at " + module.getModel().getPomFile().getAbsolutePath() + ". Unwatch..."